lib/apartment.rb

```diff
@@ -0,0 +1,32 @@
+class Apartment
+
+  attr_reader :unit_info, :renter
+
+  def initialize(unit_info = {})
+    @unit_info = unit_info
+    @renter = nil 
```

Comment:
We can store each into an instnace variable so we can clean things up using attr_reader. But this works and shows me you understand how things are coming in